### ISSUE TITLE
 Batch size setting using Resnete18

### ISSUE THREAD

**Issue by appleleaves** [larq/larq#229]
The default setting of Resnete18 use a batchsize of 1024, it takes much resources to train which I can not afford. 
Have anyone tried to use a smaller batch-size and achieve a similar performance?

**Comment by lgeiger** [comment]
I haven't tried it personally, we mainly opted for a high batch size in order to get faster training on our multi GPU setup.
To get similar training characteristics, I'd scale the learning rate together with the batch size.
I'd use the following parameters as a first try:
```python
batch_size = 256
learning_rate = 0.001
```
You can compare your training progress with https://larq.dev/api/larq_zoo/#binaryresnete18 to make sure, that you are in the right ball park.

**Comment by lgeiger** [comment]
Did the above settings work for you?

Closing, due to inactivity. @appleleaves Feel free to reopen if you need more clarification.

**Comment by pkuvmc** [comment]
Thanks a lot! I try these setting and get a similar results!
Sorry for the late reply.

**Comment by appleleaves** [comment]
Thanks a lot! I try these setting and get a similar results!
Sorry for the late reply.

**Comment by lgeiger** [comment]
> Thanks a lot! I try these setting and get a similar results!

That's great to hear 👍
